src/java.net.http/share/classes/module-info.java

```diff
@@ -25,7 +25,125 @@
+ * from being set depending on the context. This includes the "Authorization" header when the relevant
+ * HttpClient has an authenticator set. These restrictions cannot be overridden by this property.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.httpclient.bufsize}</b> (default: 16384 bytes or 16 kB)<br>
+ * The size to use for internal allocated buffers in bytes.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.httpclient.connectionPoolSize}</b> (default: 0)<br>
+ * The maximum number of connections to keep in the HTTP/1.1 keep alive cache. A value of 0 means that
+ * the cache is unbounded.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.httpclient.connectionWindowSize}</b> (default: 2^26)<br>
+ * The HTTP/2 client connection window size in bytes. The maximum size is 2^31-1. This value cannot be
+ * smaller than the stream window size.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.httpclient.disableRetryConnect}</b> (default: false)<br>
+ * Whether automatic retry of connection failures is disabled. If false, then retries are attempted
+ * (subject to the retry limit).
+ * </li>
+ * <li><p><b>{@systemProperty jdk.httpclient.enableAllMethodRetry}</b> (default: false)<br>
+ * Whether it is permitted to automatically retry non-idempotent HTTP requests.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.httpclient.enablepush}</b> (default: 1)<br>
+ * Whether HTTP/2 push promise is enabled. A value of 1 enables push promise; a value of 0 disables it.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.httpclient.hpack.maxheadertablesize}</b> (default: 16384 or 16 kB)<br>
+ * The HTTP/2 client maximum HPACK header table size in bytes.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.httpclient.HttpClient.log}</b> (default: none)<br>
+ * Enables high-level logging of various events through the Java Logging API (which is contained in the
+ * package java.util.logging). The value contains a comma-separated list of any of the following items:
+ * <ul>
+ *   <li>errors</li>
+ *   <li>requests</li>
+ *   <li>headers</li>
+ *   <li>frames</li>
+ *   <li>ssl</li>
+ *   <li>trace</li>
+ *   <li>channel</li>
+ * </ul><br>
+ * You can append the frames item with a colon-separated list of any of the following items:
+ * <ul>
+ *   <li>control</li>
+ *   <li>data</li>
+ *   <li>window</li>
+ *   <li>all</li>
+ * </ul><br>
+ * Specifying an item adds it to the HTTP client's log. For example, if you specify the following value,
+ * then the Java Logging API logs all possible HTTP Client events:<br>
+ * "errors,requests,headers,frames:control:data:window,ssl,trace,channel"<br>
+ * Note that you can replace control:data:window with all. The name of the logger is
+ * "jdk.httpclient.HttpClient", and all logging is at level INFO.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.httpclient.keepalive.timeout}</b> (default: 1200)<br>
+ * The number of seconds to keep idle HTTP/1.1 connections alive in the keep alive cache.
+ * </li>
```

Comment:
This is no longer strictly true. Also `jdk.httpclient.keepalive.timeout.h2` should be documented, see [JDK-8295649](https://bugs.openjdk.org/browse/JDK-8295649)